src/mito2/src/read/scan_region.rs

```diff
@@ -234,6 +243,10 @@ impl ScanRegion {
 
     /// Use the latest schema to build the index applier.
     fn build_index_applier(&self) -> Option<SstIndexApplierRef> {
+        if self.ignore_inverted_index {
```

Comment:
What if the index file doesn't exist? The applier or the `SstIndexApplierBuilder` might produce some warning/error logs which will be confusing.

Reply:
The IndexApplier built here and Index files are decoupled, allowing IndexApplier to apply multiple index files.

Within the Parquet Reader, if the Index is unavailable, it will not be applied:
```rust
self.file_handle.meta().inverted_index_available()
```

ref: https://github.com/GreptimeTeam/greptimedb/blob/93f28c2a375888352e1a22b8698a4e86fcdaa3fd/src/mito2/src/sst/parquet/reader.rs#L265-L275